### DISCUSSION TITLE
miner can run on the arm32 platform?

### DISCUSSION THREAD

**Discussion by philzhousad** [helium/miner#1605] [Q&A] [answered]
I want to run miner on my embedded system with arm32.  But as far as I know, miner are all run on the arm64/amd64 platforms for those known hotspots.  Does anyone know miner can run on the arm32 platform?

**Comment by meowshka** [comment]
@philzhousad we’re moving all the existing issues to the Discussions board. While your original issue will be closed, it will be open and available for collaboration in the Discussions board.

**Comment by JayKickliter** [comment, marked as answer]
It can (1) technically be compiled for arm32, but for performance reasons, it depends on CPU extensions you probably won't have on arm32. Running on arm32 will be more feasible as miner gets 'lighter'.
